packages/flex-dev-utils/src/__tests__/keytar.test.ts

```diff
@@ -23,8 +23,6 @@ describe('keytar', () => {
     jest.resetAllMocks();
     jest.resetModules();
 
-    // keytar.getService.mockImplementation(() => {});
```

Comment:
was this intentional?

Reply:
Ya, I had it commented out and forgot to remove it; just saw that in this PR